pkgs/cli/src/main.zig

```diff
@@ -235,7 +235,7 @@ pub fn main() !void {
             chain_options.num_validators = num_validators;
             // transfer ownership of the chain_options to ChainConfig
             const chain_config = try ChainConfig.init(Chain.custom, chain_options);
-            const anchorState = try sft_factory.genGenesisState(gpa.allocator(), chain_config.genesis);
+            const anchorState = try types.genGenesisState(gpa.allocator(), chain_config.genesis);
```

Comment:
I did similar thing in the #256 , and make the function in the `BeamState` struct, we may follow the style like leanSpec which make more util functions into the struct.

Reply:
@GrapeBaBa I've followed slightly different pattern with `genGenesisBlock` and `genGenesisState`, I've avoided self pointer in these fucntions to avoid first assigning the variable to undefined and then calling the function.

Reply:
I also felt it is wired before until I looked at this https://github.com/tigerbeetle/tigerbeetle/blob/main/docs/TIGER_STYLE.md#cache-invalidation, it is actual recommended approach for large struct initialization. Currently i assume the `BeamState` and `BeamBlock` large struct.